### ISSUE TITLE
Slider in SliderSettingsTile does not work with MacOs VoiceOver

### ISSUE THREAD

**Issue by hapasa** [GAM3RG33K/flutter_settings_screens#125]
When using VoiceOver on MacOs with flutter, it is impossible to change the slider value in SliderSettingsTile.

VoiceOver recognises there is a slider in there and tells how to modify the value, saying: "You are currently on a slider, inside a group. To change the value, press Control-Option and then Left Arrow or Right Arrow. ..."

However, it is seems impossible to change the value from the keyboard, only via mouse.

I'm not sure if this is some upstream flutter bug, but a vanilla flutter Slider is accessible.

Below is the code which illustrates the problem and also vanilla slider which is usable.

I do not know if this accessibility  issue is also present on iOS or Android.

Problem code:  

```
// main.dart
import 'package:flutter/material.dart';
import 'package:flutter_settings_screens/flutter_settings_screens.dart';

void main() async {
  WidgetsFlutterBinding.ensureInitialized();

  // Initialize settings
  await Settings.init(
    cacheProvider: SharePreferenceCache(),
  );

  runApp(const MyApp());
}

class MyApp extends StatelessWidget {
  const MyApp({super.key});

  @override
  Widget build(BuildContext context) {
    return MaterialApp(
      title: 'Settings Demo',
      theme: ThemeData(
        primarySwatch: Colors.blue,
      ),
      home: const SettingsPage(),
    );
  }
}

class SettingsPage extends StatefulWidget {
  const SettingsPage({super.key});

  @override
  State<SettingsPage> createState() => _SettingsPageState();
}

class _SettingsPageState extends State<SettingsPage> {
  @override
  Widget build(BuildContext context) {
    return Scaffold(
      appBar: AppBar(
        title: const Text('Settings Demo'),
      ),
      body: SafeArea(
        child: ListView(
          padding: const EdgeInsets.all(16),
          children: [
            SettingsGroup(
              title: 'Common Settings',
              children: [
                SliderSettingsTile(
                  title: 'Volume',
                  settingKey: 'key-volume',
                  defaultValue: 50,
                  min: 0,
                  max: 100,
                  step: 1,
                  leading: const Icon(Icons.volume_up),
                  onChange: (value) {
                    debugPrint('Volume: $value');
                  },
                ),
              ],
            ),
          ],
        ),
      ),
    );
  }
}
```

Working code:

```
import 'package:flutter/material.dart';

void main() {
  runApp(const MyApp());
}

class MyApp extends StatelessWidget {
  const MyApp({Key? key}) : super(key: key);

  @override
  Widget build(BuildContext context) {
    return MaterialApp(
      title: 'Flutter Slider Demo',
      theme: ThemeData(
        primarySwatch: Colors.blue,
      ),
      home: const SliderDemo(),
    );
  }
}

class SliderDemo extends StatefulWidget {
  const SliderDemo({Key? key}) : super(key: key);

  @override
  State<SliderDemo> createState() => _SliderDemoState();
}

class _SliderDemoState extends State<SliderDemo> {
  double _currentValue = 0;

  @override
  Widget build(BuildContext context) {
    return Scaffold(
      appBar: AppBar(
        title: const Text('Slider Demo'),
      ),
      body: Center(
        child: Column(
          mainAxisAlignment: MainAxisAlignment.center,
          children: [
            Text(
              'Current Value: ${_currentValue.toStringAsFixed(1)}',
              style: const TextStyle(fontSize: 24),
            ),
            const SizedBox(height: 20),
            Slider(
                value: _currentValue,
                min: 0,
                max: 100,
                divisions: 10,
                label: _currentValue.round().toString(),
                onChanged: (value) {
                  setState(() {
                    _currentValue = value;
                  });
                },
              ),
          ],
        ),
      ),
    );
  }
}

```
